### DISCUSSION TITLE
Question: Are the Masters/PhD topics mentioned in the README up for grabs?

### DISCUSSION THREAD

**Discussion by 1ntEgr8** [koka-lang/koka#421] [General]
Hi all, I'll be starting my Masters in the fall. I'm interested in contributing to Koka and was wondering if the tasks mentioned in the README are still available. If not, what are some other good avenues for getting involved?

**Comment by daanx** [comment]
Hi @1ntEgr8 -- yes, there are many things to do, and some may be good for a Master's project. If you are interested, maybe it is best to write me an email to daan at microsoft.com and explain a bit of your background and interest and we take it from there.
Thanks, -- Daan

**Comment by 1ntEgr8** [comment]
@daanx Sounds good. I've emailed you. It should be addressed from eltonp3103 at gatech dot edu

**Comment by TimWhiting** [comment]
@daanx 
Hi Daan, My professor and I also just sent you an email, it should be from: kimball@cs.byu.edu. Looking forward to potential collaboration on koka!
